modules/oc-mirror-troubleshooting-v2.adoc

```diff
@@ -0,0 +1,53 @@
+
+.Procedure
+
+. Check for server-related issues:
++
+.Example error
+[source,terminal]
+----
+[ERROR]  : [Worker] error mirroring image localhost:55000/openshift/graph-image:latest error: copying image 1/4 from manifest list: trying to reuse blob sha256:edab65b863aead24e3ed77cea194b6562143049a9307cd48f86b542db9eecb6e at destination: pinging container registry localhost:5000: Get "https://localhost:5000/v2/": http: server gave HTTP response to HTTPS client
+----
+
+.. Open the `mirroring_error_date_time.log` file in the  `working-dir/logs` folder located in the oc-mirror plugin v2 output directory.
+.. Look for error messages that typically indicate server-side issues, such as `HTTP 500` errors, expired tokens, or timeouts.
+.. Retry the mirroring process or contact support if the issue persists.
+
+. Check for incomplete mirroring of Operators:
++
+.Example error
+[source,terminal]
+----
+error mirroring image docker://registry.redhat.io/3scale-amp2/zync-rhel9@sha256:8bb6b31e108d67476cc62622f20ff8db34efae5d58014de9502336fcc479d86d (Operator bundles: [3scale-operator.v0.11.12] - Operators: [3scale-operator]) error: initializing source docker://localhost:55000/3scale-amp2/zync-rhel9:8bb6b31e108d67476cc62622f20ff8db34efae5d58014de9502336fcc479d86d: reading manifest 8bb6b31e108d67476cc62622f20ff8db34efae5d58014de9502336fcc479d86d in localhost:55000/3scale-amp2/zync-rhel9: manifest unknown
+error mirroring image docker://registry.redhat.io/3scale-amp2/3scale-rhel7-operator-metadata@sha256:de0a70d1263a6a596d28bf376158056631afd0b6159865008a7263a8e9bf0c7d error: skipping operator bundle docker://registry.redhat.io/3scale-amp2/3scale-rhel7-operator-metadata@sha256:de0a70d1263a6a596d28bf376158056631afd0b6159865008a7263a8e9bf0c7d because one of its related images failed to mirror
+error mirroring image docker://registry.redhat.io/3scale-amp2/system-rhel7@sha256:fe77272021867cc6b6d5d0c9bd06c99d4024ad53f1ab94ec0ab69d0fda74588e (Operator bundles: [3scale-operator.v0.11.12] - Operators: [3scale-operator]) error: initializing source docker://localhost:55000/3scale-amp2/system-rhel7:fe77272021867cc6b6d5d0c9bd06c99d4024ad53f1ab94ec0ab69d0fda74588e: reading manifest fe77272021867cc6b6d5d0c9bd06c99d4024ad53f1ab94ec0ab69d0fda74588e in localhost:55000/3scale-amp2/system-rhel7: manifest unknown
+----
+
+.. Check for warnings in the console or log file indicating which Operators are incomplete.
+
+If an Operator is flagged as incomplete, the image related to that Operator likely failed to mirror.
+.. Manually mirror the missing image or retry the mirroring process.
+
+. Errors related to generated cluster resources. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources such as `IDMS.yaml` and `ITMS.yaml` files, for the successfully mirrored images.
```

Comment:
I think for this second sentence, either two commas need to surround the "such as" phrase or no commas:
```suggestion
. Errors related to generated cluster resources. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources, such as `IDMS.yaml` and `ITMS.yaml` files, for the successfully mirrored images.
```
```suggestion
. Errors related to generated cluster resources. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources such as `IDMS.yaml` and `ITMS.yaml` files for the successfully mirrored images.
```